addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -8,6 +8,7 @@
 }
 
 .o_ComposerTextInput_textarea {
+    min-height: 39px;
```

Comment:
Why is this necessary now, and why 39px?

Reply:
It fixed an issue I don't have anymore. I guess other cleanup also fix this size issue.